packages/hardhat-plugin/src/utils/error-deployment-result-to-exception-message.ts

```diff
@@ -78,6 +79,14 @@ function _convertExecutionError(result: ExecutionErrorDeploymentResult) {
     sections.push(`Failures:\n\n${errorList.join("\n")}`);
   }
 
+  if (messageDetails.held) {
+    const errorList = result.held.map(
```

Comment:
Same

Reply:
I think this one is legitimate. An execution that is halted but not complete is error/failed. We need to be able to display failures/timeouts/held all at once.

Taking a look at the result type:

![image](https://github.com/NomicFoundation/ignition/assets/24030/54c70925-43e6-4435-8a21-443cc4af8087)

We could add another result type, but I think what we really want to do is rename `ExecutionErrorDeploymentResult` to something like `ExecutionHaltedDeploymentResult`?

Reply:
Sorry, I just meant the name of the variable `errorList` 😳

It could be `heldResons` or something like that. This is not important though.